### DISCUSSION TITLE
Paste text from the clipboard to search for a file

### DISCUSSION THREAD

**Discussion by drazde** [junegunn/fzf#3104] [Ideas]
I think it might be useful to be able to paste text from the clipboard.
Currently it is not possible to do this (with Ctrl+v and Shift+Ins)

**Comment by Ioulan** [comment]
`Ctrl+Shift+V`

**Reply by drazde** [reply]
> `Ctrl+Shift+V`

For me open a `Ctrl+V` open the file selected in vertical spit

**Reply by Ioulan** [reply]
> For me open a `Ctrl+V` open the file selected in vertical spit

I mean pressing `Ctrl+Shift+V` will paste text from the clipboard. I'm sorry, I think I misunderstood what you were trying to say.
Perhaps something like this the one you want to achieve?

[Kooha-2023-01-11-09-15-41.webm](https://user-images.githubusercontent.com/114978689/211696876-ff3cd556-41d1-4eb6-a471-f7f9d768902e.webm)

Pressing `Ctrl-V` will split the pane down using fzf-tmux though the previous active pane screen doesn't get clear
```bash
ctrl-v() {
  LBUFFER="$(wl-paste -n)"
  zle clear-screen
  fd --color=always --hidden | fzf-tmux -d -q "${LBUFFER}"
  zle reset-prompt
}

zle -N ctrl-v
bindkey '^V' ctrl-v
```
